src/library/Graphs/Utils.ts

```diff
@@ -191,7 +197,34 @@ export const calculatePayoutsByDay = (
       }
     }
   }
-  return payoutsByDay;
+
+  // if we don't need to take an average, just return the `payoutsByDay`.
+  if (average <= 1) {
+    return payoutsByDay;
+  }
```

Comment:
probably could get a little more clever with the logic here, but I have written it in a way which does not touch any of the existing logic, and when `average` is 0 or 1, behaves exactly the same as before.

I think the efficiency here is negligible for clarity of the PR